src/components/Login/Login.test.js

```diff
@@ -55,16 +55,20 @@ describe('Login', () => {
     });
 
     it('should call loginUser with the correct params', async () => {
-      const loginUserMock = jest.fn();
+      const loginUserMock = jest.fn()
       wrapper = shallow(<Login loginUser={loginUserMock} />)
-      const mockCurrentUser = {name: 'hill', id: 2, favorites: [], status: 'success'};
-      api.postData = jest.fn(() => mockCurrentUser)
+      const mockUser = {name: 'hill', id: 2};
+      const mockData = { data: mockUser, status: 'success'}
+      const mockCurrentUser = {
+        id: 2,
+        name: 'hill',
+        favorites: [],
+        status: 'success'
+      }
+      api.postData = await jest.fn(() => mockData)
       const mockEvent = { preventDefault: jest.fn() }
-      // await wrapper.find('form').simulate('submit', mockEvent)
       await wrapper.instance().handleSubmit(mockEvent)
-      // wrapper.setProps({ loginUser: loginUserMock })
-      await expect(wrapper.props('loginUser')).toEqual(loginUserMock);
-      expect(loginUserMock).toHaveBeenCalledWith(mockCurrentUser)
+      expect(wrapper.instance().props.loginUser).toHaveBeenCalledWith(mockCurrentUser)
```

Comment:
Thanks for figuring this out - I paired with at least 5 people and couldn't get it working!